index.html

```diff
@@ -0,0 +1,127 @@
+    <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+  </head>
+  <body>
+    <div class="container">
+      <header class="banner">
+        <div class="row">
+          <div class="eight columns">
+            <section class="logo">
+              <img src="http://i.imgur.com/5tFwpoN.png">
+              <section class="logo-text">
+                <h1> Delicious </h1>
+                <h2> THE BEST FOOD BLOG ON THE WEB </h2>
+            </section>
+          </div>
+          <div class="four columns">
+          <nav class="social-icons">
+              <ul>
+                <li><span class="icon-facebook2"></li>
+                <li><span class="icon-twitter2"></li>
+                <li><span class="icon-google-plus2"></li>
+                <li><span class="icon-camera"></li>
+                <li><span class="icon-flickr3"></li>
+                <li><span class="icon-pinterest"></li>
+                <li><span class="icon-station2"></li>
+                <li><span class="icon-mail3"></li>
+              <ul>
+            </nav>
+          </div>
+        </div>
+      </header>
+      <main>
+        <article>
+          <header class="recipe-header">
+            <div class="row">
+              <div class="ten columns">
+                <h1> Chocolate Pizza </h1>
+                <p> POSTED ON <time datetime="2013-12-15"> 15 DEC 2013 </time> / DESSERTS </p>
+              </div>
+              <div class="two columns">
+                <section class="print">
+                  <p><span class="icon-printer2"> PRINT</p>
+                </section>
+              </div>
+            </div>
+            <img class="food-pic" src="http://i.imgur.com/fB4svJW.png">
+          </header>
+          <section class="recipe-instructions">
+            <p> For the fig-swirl: Melt butter over medium heat in a saucepan. Add brown sugar and stir to dissolve. Halve all of the figs and toss in the saucepan with water and lemon juice. Cook over medium heat, stirring frequently, until you have a chunky-jammy mixture. Add salt with one or two stires, set aside and let cool completely. </p>
+            <p> Ice cream: In a small pot over medium head, combine milk, and granulated sugar until sugar is completely dissolved and the milk is just barely lukewarm. Whisk in the egg yolks. Set mixture in the fridge and wait until the fig mixture is cooled. </p>
+            <p> Using an ice cream machine, pour liquids into the frozen basin and process according to the manufacturer instructions, i.e., let spin and thicken for 20 minutes before adding mascarpone, fig jam mixture, and the nuts. Continue to process for +/- 10 minutes. Pour semi-frozen mixture into a pyrex dish or glass tupperware. Freeze for at least two hours before serving. </p>
+          </section>
+          <section class="recipe-ingredients">
+            <div class="row">
+              <div class="one-half column">
```

Comment:
I apologize for misinforming you before--after talking to Nick and Dexter and doing some more research, it is actually a good idea to put these in 'ul' elements, like so
```
<ul class="one-half column">
  <li> <label> <input type="checkbox" value="milk"> 1 1/2 cups milk </label> </li>
</ul>
```
Also, this makes them block so you can avoid the <br> tag.